src/commands/deploy.ts

```diff
@@ -204,7 +219,7 @@ export const command: Command<typeof args> = async (ctx) => {
         return;
       }
 
-      const spinnerText = stepToSpinnerText(step);
+      const spinnerText = stepToSpinnerText(step, ctx);
```

Comment:
nit: I prefer functions take `ctx` as their first argument

```suggestion
      const spinnerText = stepToSpinnerText(ctx, step);
```